Answers.md

```diff
@@ -1,9 +1,24 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
 
+  Semantic HTML is meaningful code because it allows people (developers) to more easily understand what the code is by the tags it uses and how it is written.
+
```

Comment:
Good answer, but don't forget that semantic HTML is also useful for search engine purposes as well.